cmd/policy/testdata/test_policies/policy.rego

```diff
@@ -0,0 +1,7 @@
+package org
+
+policy_name["test"]
+
+enable_rule["fail_if_not_main"] { data.meta.branch != "main" } 
```

Comment:
the rule's name is not doing what it says, as the rule rule will always fail.
the condition should be within the rule for correctness